pkg/kclient/operators.go

```diff
@@ -41,3 +48,29 @@ func (c *Client) GetCustomResourcesFromCSV(csv olm.ClusterServiceVersion) []olm.
 	// we will return a list of CRs owned by the csv
 	return csv.Spec.CustomResourceDefinitions.Owned
 }
+
+// SearchClusterServiceVersionList searches for the CSV
```

Comment:
Please add more comment here that indicates that this works with the Operators.

Reply:
done